README.md

```diff
@@ -4,25 +4,122 @@ Anaconda Recipes
+ * (32-bit) https://github.com/ContinuumIO/docker-images/blob/master/conda_builder_linux/start_cpp98_32.sh
+ * Note: Ensure the files are saved as they are stored in github. For example, if the the docker_wrapper.sh file is downloaded as docker_wrapper.sh.txt, then run: mv docker_wrapper.sh.txt  docker_wrapper.sh
+
+Make each downloaded file executable. Example: chmod +x docker_wrapper.sh
+
+In the Docker Quickstart Terminal, in the directory containing your previously downloaded files, run ./start_cpp98.sh (64-bit) or ./start_cpp98_32.sh (32-bit)
+You should see the following output: 
+
+    Welcome to the conda-builder image, brought to you by Continuum Analytics.
+
+    Binaries produced with this image should be compatible with any Linux OS
+    that is at least CentOS 5 or newer (Glibc lower bound), and anything 
+    that uses G++ 5.2 or older (libstdc++ upper bound)
+
+    GCC is: gcc (GCC) 5.2.0
+    Default C++ ABI: 4 (C++98)
+    GLIBC is: glibc 2.5
+    ld/binutils is: GNU ld (GNU Binutils) 2.26.20160125
+
+    Native arch is x86_64.  To build 32-bit code, set the ARCH environment
+        variable to 32. (-e "ARCH=32" docker argument)
+
+    The dev user (currently signed in) has passwordless sudo access.
+    miniconda (2.7) is installed at /opt/miniconda.
+    git is also available.
+    Your .gitconfig has been imported.
+    Your ssh private key has been imported for passwordless ssh.
+
+    Helpful aliases:
+    clone_recipes: clones the conda/conda-recipes repo from Github
+    clone_anaconda: clones the continuumIO/anaconda (private) repo from Github
+    anaconda_setup: clones anaconda repo and sets up continuum internal build system.
+
+We have successfully loaded the conda-builder image into our Docker container. 
+We want to have our recipes in the container. An example on how to do so : ./start_cpp98.sh -v /Users/rjain/cdf/staged-recipes:/home/dev/code
+
+This loads in our directory “staged-recipes” containing conda recipes into the docker container in /home/dev/code. We can then build our recipes simply by running: conda build <conda package> in the code directory
+
+Alternatively, the image can act as a black box that we never have to interact with. Instead, we can just use inputs and get outputs. EX: ./start_cpp98.sh -v /Users/rjain/cdf/staged-recipes:/home/dev/code -- conda build code/recipes/seaborn
+
+In order to build certain conda packages, do the following in the Docker container: 
+ * conda install mercurial
+ * Ensure conda and conda-build are up to date
+ *In addition, in the home directory there should be a .condarc file. If not, create one. The .condarc file should contain: 
+
+    channels:
+    - defaults
+    - http://bremen/test-pkgs
+    
+   add_pip_as_python_dependency: False
+
+##### Windows: 
+
+Use a Windows VM with the following specifications: 
 
-Windows:
   * 64-bit Windows 10
   * VS 2008, 2010, 2015 installed
```

Comment:
VS 2008 Pro (or VC compiler for Python 2.7), VS 2010 Pro (or Windows 7.1 SDK)

Note that for both VS2008 Pro, you must explicitly enable the 64-bit compiler during setup.  During VS 2015 setup, you must explicitly enable native build tools, or else you get no C++ compiler.